tests/test_us_verification.py

```diff
@@ -8,11 +8,25 @@ def setUp(self):
 
     def test_us_verification(self):
         addr = lob.USVerification.create(
-            primary_line='185 Berry St Ste 6600',
+            primary_line='deliverable',
```

Comment:
question for clarity: How does this return deliverable?

Reply:
Since the value is in the verification itself, we cannot give it for free in test mode. We've recently changed the way we do deliverabilities in _test_ mode to allow customers to fully test all possible values that the API could return. In the latest API Version (2018-03-01), providing a primary line of `deliverable` will return a deliverable address example. You can find more info [here](https://lob.com/docs#us-verification-test-environment).